.travis/mypy

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
```

Comment:
As always I question the wisdom of writing shell scripts rather than Python scripts :)

Reply:
I hear you, and I've thought of doing this in Python, actually.  This was expedient, and I'm a little bit nervous about using Python in the build pipeline, but my reasons for that probably don't really apply here.  That said, doing so would add very little value here; this is a pretty simple script.